arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-rollup-chain.md

```diff
@@ -0,0 +1,238 @@
+---
+title: 'How to Deploy a Rollup Chain Using the Orbit SDK'
+sidebar_label: 'Rollup Orbit Deployment'
+description: 'How to deploy a Rollup chain using the Orbit SDK '
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 1
+---
+
+This section explains how to use the Orbit SDK to deploy a <a data-quicklook-from="arbitrum-rollup-chain">`Rollup Orbit chain`</a>:
+
+###### For those who prefer diving directly into coding without an extensive tutorial, we recommend exploring the ["create-rollup-eth"  example]( https://github.com/OffchainLabs/arbitrum-orbit-sdk/blob/main/examples/create-rollup-eth/index.ts) 
+
+The main benefit of the Orbit SDK lies in facilitating the deployment and fine-tuning of Orbit chains core Smart-Contracts. 
+
+These contracts are deployed on <a data-quicklook-from="parent-chain">`parent chains`</a>, they are:
+
+- <a data-quicklook-from="bridge">Bridge contracts</a>
+- Rollup contracts
+- Contracts handling <a data-quicklook-from="fraud-proof">fraud proofs</a> 
+
+Core contracts are the backbone of Arbitrum's <a data-quicklook-from="arbitrum-nitro">Nitro stack</a>, ensuring its robust and efficient operation. You can explore their code in the [nitro-contracts GitHub repository](https://github.com/OffchainLabs/nitro-contracts). 
+
+
+To streamline the deployment process and make it more efficient, we've developed a key contract called [`RollupCreator` contract](https://github.com/OffchainLabs/nitro-contracts/blob/main/src/rollup/RollupCreator.sol). This contract has two primary functions:
+
+1. [setTemplates](https://github.com/OffchainLabs/nitro-contracts/blob/acb0ef919cce9f41da531f8dab1b0b31d9860dcb/src/rollup/RollupCreator.sol#L63C14-L63C26): specifies which versions of core contract should be deployed and ensures your Orbit chain runs the latest, most efficient contracts versions.
```

Comment:
I wouldn't add `setTemplates` here, since this is a function only called by us to set new templates. The chain deployer doesn't need to use this function (or even worry about it)

Reply:
Noted! Thank you. I must admit I was not too sure about this one.

Reply:
Merged manually.